### DISCUSSION TITLE
MailService: add "ReplyTo" seems to not working

### DISCUSSION THREAD

**Discussion by emulienfou** [directus/directus#5942] [Archive]
Hello there, I'm trying to add a `reply-to` header in the email I'm sending bu this seems to not working.
I tried 2 approaches according to the documentation of nodemailer here: https://nodemailer.com/message/

First one using `replyTo`:
```js
await mailService.send({
        from: process.env.EMAIL_TO,
        to: process.env.EMAIL_TO,
        replyTo: req.body.email,
        subject: process.env.EMAIL_SUBJECT,
        template: { name: 'contact', data: req.body }
      })
```

Second one, using `headers` directly
```js
await mailService.send({
        from: process.env.EMAIL_TO,
        to: process.env.EMAIL_TO,
        headers: {'reply-to': req.body.email},
        subject: process.env.EMAIL_SUBJECT,
        template: { name: 'contact', data: req.body }
      })
```

No **reply-to** header is visible in the email I received

**Comment by rijkvanzanten** [comment]
What version of Directus are you using?

**Reply by rijkvanzanten** [reply]
FWIW, everything in that object is passed to nodemailer's `sendMail` directly, so we might have to debug this outside of Directus 🤔

**Reply by emulienfou** [reply]
Using the ** Directus 9.0.0-rc.62** version.
What I was thinking about, I may have  better chances to open and issue in the nodemailer repository

**Reply by rijkvanzanten** [reply]
Ah no never mind. You have to update Directus 🙂 

That passing to nodemailer was introduced in rc.65 👍🏻

**Reply by emulienfou** [reply]
Okay testing thanks

**Reply by emulienfou** [reply]
Fixed thanks @rijkvanzanten
